hypothesis-ruby/Rakefile

```diff
@@ -24,7 +24,9 @@ begin
 rescue LoadError
 end
 
-HelixRuntime::BuildTask.new
+build_task = HelixRuntime::BuildTask.new
+
+build_task.build_root = '..'
```

Comment:
Is this meant to be the root of the repo? Could you use `git rev-parse --show-toplevel` like we do in the build tooling?